rust/arrow/src/array.rs

```diff
@@ -761,6 +800,26 @@ mod tests {
         let _slice = arr.value_slice(0, 4);
     }
 
+    #[test]
+    fn test_int32_fmt_debug() {
+        let buf = Buffer::from(&[0, 1, 2, 3, 4].to_byte_slice());
+        let arr = Int32Array::new(5, buf, 0, 0);
+        assert_eq!(
+            "PrimitiveArray<Int32> { len: 5, values: [0, 1, 2, 3, 4] }",
+            format!("{:?}", arr)
+        );
+    }
+
+    #[test]
+    fn test_boolean_fmt_debug() {
```

Comment:
May you please add tests for null values, I don't think you handle them because getting a null value returns invalid data.

Reply:
Will do! Thanks